addon-test-support/-private/compatibility.js

```diff
@@ -57,6 +57,8 @@ if (require.has('ember-test-helpers/wait')) {
   // conditions `ember-test-helpers/wait` can still be in the pending state
   // at this point, so its exports are still undefined.
   waitFn = (...args) => require('ember-test-helpers/wait').default(...args);
+} else if (require.has('@ember-test/helpers')) {
+  waitFn = (...args) => require('@ember-test/helpers').wait(...args);
```

Comment:
shouldn't this be `@ember/test-helpers` not `@ember-test/helpers`?

https://github.com/emberjs/ember-test-helpers/blob/master/API.md#waitfor

`@ember-test/helpers` doesn't exist:
![image](https://user-images.githubusercontent.com/199018/134014613-5169a0f2-3ede-49f0-afd5-168cb73255db.png)

Reply:
It was fixed in a follow up https://github.com/san650/ember-cli-page-object/commit/75a9e8b2dff49b4c513e61422215c5ca168b33d5#diff-d52aad060d2deba9d04ba86013c19734ea5e79e60b7539d2bd563b561288e7b3